<?xml version="1.0" encoding="UTF-8"?>
<testsuite hostname="VAIO-VAIO" name="testSuiteB.TestB2" tests="3" failures="1" timestamp="12 Oct 2015 14:16:54 GMT" time="0.008" errors="0">
  <testcase name="testB2_1" time="0.0" classname="testSuiteB.TestB2"/>
  <testcase name="testB2_2" time="0.001" classname="testSuiteB.TestB2">
    <failure type="java.lang.AssertionError" message="failed test case ">
      <![CDATA[java.lang.AssertionError: failed test case 
	at testSuiteB.TestB2.testB2_2(TestB2.java:16)
... Removed 26 stack frames]]>
    </failure>
  </testcase>
  <testcase name="testB2_3" time="0.0" classname="testSuiteB.TestB2"/>
</testsuite>
